google-cloud-spanner/src/main/java/com/google/cloud/spanner/CompositeTracer.java

```diff
@@ -109,7 +108,7 @@ public void attemptCancelled() {
   }
 
   @Override
-  public void attemptFailed(Throwable error, Duration delay) {
+  public void attemptFailed(Throwable error, org.threeten.bp.Duration delay) {
```

Comment:
qq, what changed here?

Reply:
https://github.com/googleapis/java-spanner/pull/3495/files#diff-504b19ab966dfdfb231d51ebf483b78fc52af98e608f3b081e094c96a80b38f1R117-R122 shows that the duration method was already added. I just modified the signature to make the package name explicit

Reply:
Ok gotcha. Do you think we can migrate this to `attemptFailedDuration`? https://github.com/googleapis/sdk-platform-java/blob/cf32e72d83c06d1d36e1eea01afc37bee5766455/gax-java/gax/src/main/java/com/google/api/gax/tracing/ApiTracer.java#L126?

Reply:
Done